gamemodes/terrortown/gamemode/shared/hud_elements/tttinfopanel/old_ttt_info.lua

```diff
@@ -11,7 +13,9 @@ DEFINE_BASECLASS(base)
 HUDELEMENT.Base = base
 
 if CLIENT then
-	local hudTeamicon = CreateClientConVar("ttt2_base_hud_teamicon", "1")
+	---
+	-- @realm client
+	local hudTeamicon = CreateConVar("ttt2_base_hud_teamicon", "1", FCVAR_ARCHIVE)
```

Comment:
are convars detected if the file is ignored?

Reply:
Nope

Reply:
then you should probably change this